nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -318,17 +318,24 @@ void reportInitErrorOnField(Symbol symbol, VisitorState state, Description.Build
       return;
     }
     Tree tree = getTreesInstance(state).getTree(symbol);
+
+    String fieldName = symbol.toString();
+
+    if (symbol.enclClass().getNestingKind().isNested()) {
+      fieldName = symbol.enclClass().getSimpleName() + "." + symbol;
```

Comment:
If there are multiple levels of nesting this will just give the immediate enclosing class.  It seems we would potentially want all of the enclosing classes included?  @lazaroclapp?

Reply:
Honestly, I can go either way here. Error Prone already prints the file name, and fully qualified names are easy to just gloss over mentally. 

Maybe the right thing is `getQualifiedName` and then only keep everything after the last `.`? Should be `Foo$Bar$Baz`, but not `com.example.a.b.c.d.Foo$Bar$Baz` ;)  Is that safe? (Also note that both `getSimpleName()` and `getQualifiedName()` can return empty string and maybe even `null` for anonymous inner classes, so I'd want a test that those don't crash NullAway at least)

Reply:
Everything after the last `.` sounds good to me

Reply:
It would also be good to get a test with multiple levels of nesting to make sure it's doing what we expect